setup/system.sh

```diff
@@ -144,6 +144,10 @@ pollinate  -q -r
 
 # Between these two, we really ought to be all set.
 
+echo 'Launching SSH public key creation...'
+
+nohup ssh-keygen -t rsa -b 2048 -a 100 -f /root/.ssh/id_rsa_miab -N '' -q 2>&1 >/dev/null
```

Comment:
well, I'm not sure what can be done, because afaict, the `2>&1 > /dev/null` are passed to the program called by `nohup`, so I'm not sure how to redirect `nohup`'s output. And because that task is quite lengthy and it's not necessary to wait for it to be finished to get on with setup, it's a good idea to run it with nohup. If you think of an alternative.